02.calendar/cal.js

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env node
+
+import dayjs from "dayjs";
+import "dayjs/locale/ja.js";
+import timezone from "dayjs/plugin/timezone.js";
+import utc from "dayjs/plugin/utc.js";
+import minimist from "minimist";
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+dayjs.tz.setDefault("Asia/Tokyo");
+
+function main() {
```

Comment:
```suggestion
main()

function main() {
```

JSだと関数の巻き上げにより、宣言する手前でmainを呼び出せるはずです